src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -38,14 +51,34 @@
  */
 public class UnsolvedSymbolVisitor extends ModifierVisitor<Void> {
 
+  /**
+   * The parent class of this current class file. If there is no parent class, then the value of
+   * this variable is an empty string
```

Comment:
I think I will choose to clarify the javadoc for this field. I will change it to "The unsolved parent class of this current class file.", so that people will understand that this field is used to create synthetic parent class if needed.